src/harness/testcases/WINNF_FT_S_IPR_testcase.py

```diff
@@ -1098,6 +1098,7 @@ def generate_IPR_7_default_config(self, filename):
         'sasTestHarnessConfigs': [sas_test_harness_config],
         'domainProxies': [domain_proxy],
         'portalDpa': portal_dpa,
+        'runEarlyCpas': False
```

Comment:
Should we make the default True?  I run it both ways and it work and the impact (time) is minimal if the flag is set to True.  I could not tell from the recommended WG4 changes what the plan was for ITS to do first

Reply:
As far as I know, there is no WG4 preference. It was a bit cleaner to code it this way:

```if flag exists and flag=True...```

vs.

```if flag doesn't exist or (flag exists and flag=True)...```

hence I went with the first one. (If I'm honest, it's that plus personal preference for the default to be 'false.')